index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Page</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/css/bootstrap.min.css"
+    integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+  <div class="container">
+    <header>
+      <h1>Reddit</h1>
+    </header>
+  </div>
+  <main class="container">
+    <hr>
+    <div class="comment-btns">
+      <span class="rm-comments-btn">remove</span>
+      <span class="show-comments-btn">comments:</span>
+    </div>
+    <span class="new-comments">I'm a comment - Posted By: Jeff
+      <br></span>
+    <div class="comment-btns">
+      <hr>
+      <span class="rm-comments-btn">remove</span>
+      <span class="show-comments-btn">comments:</span>
+    </div>
+    <span class="new-comments">Here's a comment - Posted By: Jeff <br>
+    </span>
+    <hr>
+    <input class="post" type="text" placeholder="Post Text" />
+    <input class="name" type="text" placeholder="Your Name" />
+    <ul class="nav nav-pills">
+      <li role="presentation" class="active"><a href="#">Submit</a></li>
```

Comment:
never use ul or lists for buttons, use the button tag and you have 'btn btn-primary' bootstrap classes